src/main/java/mdlaf/themes/AbstractMaterialTheme.java

```diff
@@ -1090,7 +1090,7 @@ public boolean getEnableArrowScrollBar() {
     return false;
   }
 
-  // get jslider
+  // get JSlider
```

Comment:
This comment can be removed. It have no sense here

Reply:
Thanks, that's now removed!